src/System.IO.FileSystem.AccessControl/src/System/Security/AccessControl/FileSecurity.cs

```diff
@@ -319,8 +319,8 @@ internal FileSystemSecurity(bool isContainer)
         {
         }
 
-        internal FileSystemSecurity(bool isContainer, string name, AccessControlSections includeSections, bool isDirectory)
```

Comment:
If you agree, I renamed `name` to `fileName` for clarity. Maybe I could rename it to `path`, because we can receive both file paths and directory paths.